public/doc/openapi-3-v1.0.0.json

```diff
@@ -1384,6 +1426,177 @@
+          "Icon"
+        ],
+        "summary": "Create an Icon",
+        "description": "Creates an Icon from the specified parameters",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Icon"
+              }
+            }
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The newly created Icon",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Icon"
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          }
+        }
+      }
+    },
+    "/icons/{id}": {
+      "get": {
+        "tags": [
+          "Icon"
+        ],
+        "summary": "Fetch an Icon by ID",
+        "description": "Fetch an Icon by ID",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/ID"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Icon matching the Icon ID",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Icon"
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "Icon not found"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Icon"
+        ],
+        "summary": "Edit an existing Icon",
+        "description": "Edits Icon specified by the given ID.",
+        "parameters": [
+          {
+            "$ref": "#/components/parameters/ID"
+          }
+        ],
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Icon"
+              }
+            }
+          },
+          "description": "Parameters needed to update an Icon",
+          "required": true
+        },
+        "operationId": "updateIcon",
+        "responses": {
+          "200": {
+            "description": "Return the updated Icon object",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/Icon"
+                }
+              }
+            }
+          },
+          "401": {
+            "$ref": "#/components/responses/Unauthorized"
+          },
+          "403": {
+            "$ref": "#/components/responses/Forbidden"
+          },
+          "404": {
+            "description": "The Icon was not found"
```

Comment:
Should this be standardized to the other descriptions which say `"Icon not found."`?